database/src/main/resources/db/migration/V1_23__Add_tx_history_tables_and_drop_views.sql

```diff
@@ -0,0 +1,44 @@
+DROP MATERIALIZED VIEW IF EXISTS block_cache_tx_history_day;
+DROP MATERIALIZED VIEW IF EXISTS block_cache_tx_history_hour;
+
+-- HOURLY count of Tx history
+CREATE TABLE IF NOT EXISTS block_cache_hourly_tx_counts
+(
+    block_timestamp TIMESTAMP NOT NULL,
```

Comment:
Add `id` on here, I know its essentially useless, but itll make the entities and queries much happier

Reply:
It is an intentional anti-pattern. If we add it, my concern is that it will grow very fast because of the `delete` and `re-insert` operation of the aggregate counts. To avoid issues with queries, I'm using '.silce' to select only the desired columns.

Reply:
I was gonna suggest that as an alternative.

Reply:
I'll give that a try. If it fails, I'll switch to using the DSL API